index.html

```diff
@@ -160,10 +160,23 @@ <h2>
       </ul>
       <p>
         The details of how to fulfill a payment request for a given <a>payment
-        method</a> are handled by <dfn data-lt="payment handler">payment
-        handlers</dfn>. In this specification, these details are left up to the
-        <a>user agent</a>, but future specifications may expand on the
-        processing model in more detail.
+        method</a> is an implementation detail of a <dfn data-lt=
+        "payment handlers">payment handler</dfn>. Additionally, each payment
+        handler defines a <dfn>can make payment</dfn> algorithm. The details of
+        how a payment handler determines wether it, or the user, can
+        potentially "make a payment" is also an implementation detail of a
+        payment handler. For an example, see the <a data-cite=
+        "payment-method-basic-card#can-makepayment">can make payment</a>
+        algorithm of [[payment-method-basic-card]].
+      </p>
+      <p>
+        A payment handler also defines the <a>steps to respond to a payment
+        request</a>, which results with an object, or <a>dictionary</a>, that a
+        a merchant uses to process or validate the transaction. The structure
+        of this object is specific for each <a>payment handler</a>. For an
+        example of such an object, see the <code><a data-cite=
+        "payment-method-basic-card#dom-basiccardresponse">BasicCardResponse</a></code>
+        dictionary of [[payment-method-basic-card]].
```

Comment:
Please change "The structure of this object is specific for each <a>payment handler</a>" to "The structure
of this object is specific for each <a>payment method</a>."